include/zephyr/arch/arm/aarch32/cortex_a_r/scripts/linker.ld

```diff
@@ -202,6 +202,8 @@ SECTIONS
 
 #include <zephyr/linker/kobject-rom.ld>
 
+        *(.shared_isr_args)
+        *(.shared_isr)
```

Comment:
How did you decide where to put those symbols? Some platforms do not have much freedom in that regard. See for example 8c4f98d263865d00199fc9b5f7870a0f134c634b.

Reply:
I placed them after all other sections to avoid shifting of the address.

Reply:
This is wrong for several reasons. A couple here.
1. These could be the latest sections when using this vanilla linker script. But some boards can add some custom sections after, so those are going to be shifted.
2. Some platforms require the ISR to be in some specific sections or delimited by special symbols (see 8c4f98d263865d00199fc9b5f7870a0f134c634b).

The whole point of having `arch/common/isr_tables.c` is indeed having to reserve the space in advance so that the symbols are not going to be shifted. Otherwise we could have moved the location of the tables in the first place.

Reply:
Thanks for your patience, I'm new to this part of Zephyr. I've moved the shared ISRs and arguments to the .last_section. If this is still incorrect, please advise on the proper way to implement this.